actions/setup-gap/action.yml

```diff
@@ -182,6 +213,11 @@ runs:
         AWS_SECRET_ACCESS_KEY: ${{ env.AWS_SECRET_ACCESS_KEY }}
         AWS_SESSION_TOKEN: ${{ env.AWS_SESSION_TOKEN }}
       run: |
+        DUPLICATE_AUTH_HEADER_FLAG=""
+        if [ "${{ inputs.duplicate-authorization-header }}" == "true" ]; then
+          DUPLICATE_AUTH_HEADER_FLAG="--duplicate-headers Authorization"
```

Comment:
is the flag called `--duplicate-headers`?

Reply:
Yes - https://github.com/awslabs/aws-sigv4-proxy/commit/5770ebf7f95b58bad15546a2d1827040d61ec611